docs/enketo.md

```diff
@@ -0,0 +1,61 @@
+# Running Enketo for development
+
+Enketo provides a web-based form submission UI, which is used in ODK Central for previewing forms, collecting new Submissions through Public Access Links, and editing Submissions. It is not strictly required for running ODK Central Frontend, but there are an increasing number of new features (including editing Submissions) that make use of Enketo. The following configuration is only for a development environment. The production ODK Central stack already includes Enketo with no additional configuration.
+
+Enketo runs as a Node.js server and caches intermediate representations of forms in Redis. ODK Central Backend stores information for managing and launching Enketo forms from ODK Central Frontend, so all three must be configured together.
+
+- Clone [enketo-express](https://github.com/enketo/enketo-express) and read the [Manual Installation Instructions](https://github.com/enketo/enketo-express/blob/master/tutorials/00-getting-started.md#manually). Specifically:
+	* Install Redis. (Node.js will already be installed to run ODK Central Frontend and Backend.) Enketo will expect Redis on the default port of `6379`.
+	* Install dependencies with `npm install`.
+- Configure Enketo. A minimal `config/config.json` for Enketo looks like:
+
+```
+{
+    "port": "8005",
+    "linked form and data server": {
+        "server url": "localhost:8989",
+        "api key": "enketorules",
+        "authentication": {
+            "allow insecure transport": "true"
+        }
+    },
+    "base path": "-",
+    "redis": {
+        "cache": {
+            "port": "6379"
+        }
+    }
+}
+```
+
+- Configure ODK Central Backend. Add the following to `config/default.json`.
+
+```
+    "enketo": {
+      "url": "http://localhost:8005/-",
+      "apiKey": "enketorules"
+    },
+```
+
+- Configure Nginx in ODK Central Frontend. The following lines have already been added to `nginx.conf`. They create a reverse proxy for the Enketo server in the same way as the reverse proxy to the ODK Central Backend server. 
```

Comment:
Nit: Now that the `nginx` config includes this, maybe this step could go last and the first sentence could go? Nice to have the explanation but this implies there's an action to take.